client/src/Movies/SavedList.js

```diff
@@ -10,9 +11,13 @@ export default class SavedList extends Component {
       <div className="saved-list">
         <h3>Saved Movies:</h3>
         {this.props.list.map(movie => (
-          <span className="saved-movie">{movie.title}</span>
+          <NavLink activeClassName='selected' to={`/movies/${movie.id}`}
+          activeStyle={{
+            fontWeight: 'bold',
+            color: 'teal', 
+          }}><span key={movie.id} className="saved-movie">{movie.title}</span></NavLink>
```

Comment:
whats stopping me from saving one movie, twice ? Therefore our key will not be unique